── 1. Failure: Failure:1 (@tests.R#8)  ─────────────────────────────────────────
Failure has been forced

── 2. Failure: Failure:2a (@tests.R#12)  ───────────────────────────────────────
Failure has been forced

── 3. Failure: Failure:2b (@tests.R#15)  ───────────────────────────────────────
FALSE isn't true.

── 4. Failure: Failure:loop (@tests.R#20)  ─────────────────────────────────────
`i` not equal to 2.
1/1 mismatches
[1] 1 - 2 == -1

── 5. Error: Error:1 (@tests.R#28)  ────────────────────────────────────────────
stop
1: stop("stop") at reporters/tests.R:28

── 6. Error: Error:3 (@tests.R#42)  ────────────────────────────────────────────
!
1: f() at reporters/tests.R:42
2: g() at reporters/tests.R:33
3: h() at reporters/tests.R:36
4: stop("!") at reporters/tests.R:39

── 7. Error: Recursion:1 (@tests.R#51)  ────────────────────────────────────────
This is deep
1: f(25) at reporters/tests.R:51
2: f(x - 1) at reporters/tests.R:49
3: f(x - 1) at reporters/tests.R:49
4: f(x - 1) at reporters/tests.R:49
5: f(x - 1) at reporters/tests.R:49
6: f(x - 1) at reporters/tests.R:49
7: f(x - 1) at reporters/tests.R:49
8: f(x - 1) at reporters/tests.R:49
9: f(x - 1) at reporters/tests.R:49
10: f(x - 1) at reporters/tests.R:49
...
18: f(x - 1) at reporters/tests.R:49
19: f(x - 1) at reporters/tests.R:49
20: f(x - 1) at reporters/tests.R:49
21: f(x - 1) at reporters/tests.R:49
22: f(x - 1) at reporters/tests.R:49
23: f(x - 1) at reporters/tests.R:49
24: f(x - 1) at reporters/tests.R:49
25: f(x - 1) at reporters/tests.R:49
26: f(x - 1) at reporters/tests.R:49
27: stop("This is deep") at reporters/tests.R:49

══ testthat results  ═══════════════════════════════════════════════════════════
OK: 7 SKIPPED: 3 FAILED: 7
1. Failure: Failure:1 (@tests.R#8) 
2. Failure: Failure:2a (@tests.R#12) 
3. Failure: Failure:2b (@tests.R#15) 
4. Failure: Failure:loop (@tests.R#20) 
5. Error: Error:1 (@tests.R#28) 
6. Error: Error:3 (@tests.R#42) 
7. Error: Recursion:1 (@tests.R#51) 

